quick-start/src/main/resources/application.properties

```diff
@@ -1,10 +1,11 @@
 #Spring embedded tomcat config. Does not apply when war file is deployed on external Tomcat.
-server.port=8080
+server.port=8081
 server.contextPath=/
 spring.thymeleaf.cache=true
 logging.level.com.marklogic.hub: DEBUG
 mlHost.default=localhost
-mlRestPort.default=8010
+mlStagingRestPort.default=8010
+mlFinalRestPort.default=8011
 mlUsername.default=admin
```

Comment:
We decided to keep it crazy simple. It's Quick Start and thus it's more of a demo of the hub.